src/PairEvaluatorPerturbedLennardJones.h

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2024, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+#ifndef AZPLUGINS_PAIR_EVALUATOR_PERTURBED_LENNARD_JONES_H_
+#define AZPLUGINS_PAIR_EVALUATOR_PERTURBED_LENNARD_JONES_H_
+
+#include "PairEvaluator.h"
+
+#ifdef __HIPCC__
+#define DEVICE __device__
+#define HOSTDEVICE __host__ __device__
+#else
+#define DEVICE
+#define HOSTDEVICE
+#endif
+
+namespace hoomd
+    {
+namespace azplugins
+    {
+namespace detail
+    {
+//! Define the parameter type used by this pair potential evaluator
+struct PairParametersPerturbedLennardJones : public PairParameters
+    {
+#ifndef __HIPCC__
+    PairParametersPerturbedLennardJones() : sigma_6(0), epsilon_x_4(0), lam(0), rwcasq(0) { }
+
+    PairParametersPerturbedLennardJones(pybind11::dict v, bool managed = false)
+        {
+        auto sigma(v["sigma"].cast<Scalar>());
+        auto epsilon(v["epsilon"].cast<Scalar>());
+
+        const Scalar sigma_2 = sigma * sigma;
+        const Scalar sigma_4 = sigma_2 * sigma_2;
+        sigma_6 = sigma_2 * sigma_4;
+
+        epsilon_x_4 = Scalar(4.0) * epsilon;
+        lam = v["lam"].cast<Scalar>();
+        rwcasq = pow(Scalar(2.), 1. / 3.) * sigma_2;
+        }
+
+    pybind11::dict asDict()
+        {
+        pybind11::dict v;
+        v["sigma"] = pow(sigma_6, 1. / 6.);
+        v["epsilon"] = epsilon_x_4 / Scalar(4.);
+        v["lam"] = lam;
+        return v;
+        }
+#endif // __HIPCC__
+
+    Scalar sigma_6;     //!< The coefficient for 1/r^12
+    Scalar epsilon_x_4; //!< The coefficient for 1/r^6
+    Scalar lam;         //!< Controls the attractive tail, between 0 and 1
+    Scalar rwcasq;      //!< The square of the location of the LJ potential minimum
+    }
+#if HOOMD_LONGREAL_SIZE == 32
+    __attribute__((aligned(16)));
+#else
+    __attribute__((aligned(32)));
+#endif
```

Comment:
Alignment should be a power of two and based on the size of the data. Usually 8, 16, or 32 bytes is appropriate.